js/popup.js

```diff
@@ -0,0 +1,37 @@
+import {BUILDING_TYPES} from './constants.js';
+import {
+  fillTextContent,
+  getPriceDescription,
+  getCapacityDescription,
+  getCheckDescription,
+  filterFeatures,
+  fillLinks,
+  fillAvatar
+} from './utils.js';
+
+/**
+ * Создает и возвращает модальное окно с переданными значениями
+ * @param {object} advertisement - предложение об аренде
+ * @param {HTMLElement} template - шаблон модального окна
+ * @param {string} selector - селектор модального окна
+ * @return {Node} - модальное окно с заполненными значениями.
+ */
+const createPopup = (advertisement, template, selector) => {
+  const popupTemplate = template.querySelector(selector);
+  const popup = popupTemplate.cloneNode(true);
+
+  fillTextContent(popup, '.popup__title', advertisement.offer.title);
```

Comment:
я бы сделал дом функцию поиска ноды в контейнере и сделал бы вот так 
```suggestion
  fillTextContent(findNode(popup, '.popup__title'), advertisement.offer.title);
```